rajawali/src/main/java/rajawali/renderer/RajawaliRenderer.java

```diff
@@ -132,20 +147,33 @@ public RajawaliRenderer(Context context) {
 		mContext = context;
 		mFrameRate = getRefreshRate();
 		mScenes = Collections.synchronizedList(new CopyOnWriteArrayList<RajawaliScene>());
-		mSceneQueue = new LinkedList<AFrameTask>();
-		mSceneCachingEnabled = true;
-		mSceneInitialized = false;
+        mRenderTargets = Collections.synchronizedList(new CopyOnWriteArrayList<RenderTarget>());
+        mSceneQueue = new LinkedList<AFrameTask>();
+        mSceneCachingEnabled = true;
+        mSceneInitialized = false;
+
+        mLoaderThreads = new SparseArray<ModelRunnable>();
+        mLoaderCallbacks = new SparseArray<IAsyncLoaderCallback>();
```

Comment:
Are the generic definitions required here? If not we should aim to start removing the J7 notices about it.